.github/workflows/build_frontend.yaml

```diff
@@ -3,7 +3,6 @@ name: Frontend
 on:
   pull_request:
     branches:
-      - master
```

Comment:
removing `master` means this workflow will not run.  `required` workflows must run (even if there is nothing to do).

Reply:
Ah, I see what you were getting at now. Yeah there's a conundrum here with required workflows...

Reply:
build_frontend has been reverted back to conditional-based behavior